Framework/Crystal/src/CentroidPeaks.cpp

```diff
@@ -357,5 +336,23 @@ int CentroidPeaks::findPixelID(std::string bankName, int col, int row) {
   }
 }
 
+void CentroidPeaks::removeEdgePeaks(
+    Mantid::DataObjects::PeaksWorkspace *peakWS) {
+  int Edge = getProperty("EdgePixels");
+  std::vector<int> badPeaks;
+  for (int i = int(peakWS->getNumberPeaks()) - 1; i >= 0; --i) {
+    // Get a direct ref to that peak.
+    auto &peak = peakWS->getPeak(i);
+    int col = peak.getCol();
+    int row = peak.getRow();
+    std::string bankName = peak.getBankName();
```

Comment:
Take this by const ref instead as the compiler will make a copy of this:
`const std::string &bankName`